istio/README.md

```diff
@@ -112,6 +128,19 @@ See [service_checks.json][16] for a list of service checks provided by this inte
 
 ## Troubleshooting
 
+### Invalid chunk length error
+If you see the following error on OpenMetricsBaseCheck (V1) implementation of Istio (Istio integration version `3.13.0` or older):
+
+    ```python
+      Error: ("Connection broken: InvalidChunkLength(got length b'', 0 bytes read)", 
+      InvalidChunkLength(got length b'', 0 bytes read))
+    ```
+
+You can use the Openmetrics V2 implementation of the Istio integration to resolve this error.
+
+Note: you must upgrade to at minimum Agent `7.31.0` and Python 3. See above Configuration section on enabling Openmetrics V2.
```

Comment:
can you link the Configuration section?

```suggestion
Note: you must upgrade to at minimum Agent `7.31.0` and Python 3. See above [Configuration](#configuration) section on enabling Openmetrics V2.
```

that should work for markdown.

Reply:
```suggestion
Note: you must upgrade to at minimum Agent `7.31.0` and Python 3. See the [Configuration](#configuration) section on enabling Openmetrics V2.
```